src/Conference/UpdateConference.jsx

```diff
@@ -17,6 +15,8 @@ const UpdateForm = ({
   error
 }) => {
   const history = useHistory();
+  const [cookies] = useCookies([cookieName]);
+  const token = cookies.sessionToken.token;
```

Comment:
Is there a possibility that someone who's not logged in lands on this route i.e. `conferences/1/edit`? therefore this sessionToken might be undefined and this may blow up. so might need to do a ternary check. Similarly I would look at doing a redirect if they land on this route in your conference container

Reply:
ok yes that's true! I'll keep a note of that for one of the clean up tickets